reference/3.0/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md

```diff
@@ -115,209 +142,406 @@ ProviderLatency                : 1000
 ProviderControlGuid            :
 ```
 
-This command gets an object that represents the classic Setup log on the local computer.
-The object includes useful information about the log, including its size, event log provider, file path, and whether it is enabled.
+The `Get-WinEvent` cmdlet uses the **ListLog** parameter to specify the **Setup** log. The object is
+sent down the pipeline to the `Format-List` cmdlet. `Format-List` uses the **Property** parameter
+with the asterisk (`*`) wildcard to display each property.
 
 ### Example 3: Get event logs from a server
-```
-PS C:\> Get-WinEvent -ListLog * -ComputerName Server01 |Where-Object {$_.RecordCount}
+
+This command only gets event logs on the local computer that contain events. It's possible for a
+log's **RecordCount** to be null or zero. The example uses the `$_` variable. For more information,
+see [about_Automatic_Variables](../Microsoft.PowerShell.Core/about/about_automatic_variables).
+
+```PowerShell
+Get-WinEvent -ListLog * -ComputerName localhost | Where-Object { $_.RecordCount }
 ```
 
-This command gets only event logs on the Server01 computer that contain events.
-Many logs might be empty.
+```Output
+LogMode   MaximumSizeInBytes RecordCount LogName
+-------   ------------------ ----------- -------
+Circular            15532032       14546 Application
+Circular             1052672         117 Azure Information Protection
+Circular             1052672        2990 CxAudioSvcLog
+Circular             1052672           9 MSFTVPN Setup
+Circular             1052672         282 OAlerts
+```
 
-The command uses the **RecordCount** property of the **EventLogConfiguration** object that **Get-WinEvent** returns when you use the *ListLog* parameter.
+The `Get-WinEvent` cmdlet gets log information from the computer. The **ListLog** parameter uses the
+asterisk (`*`) wildcard to display information about each log. The **ComputerName** parameter
+specifies to get the logs from the local computer, **localhost**. The objects are sent down the
+pipeline to the `Where-Object` cmdlet. `Where-Object` uses `$_.RecordCount` to return only logs that
+contain data. `$_` is a variable that represents the current object in the pipeline. **RecordCount**
+is a property of the object with a non-null value.
 
 ### Example 4: Get event logs from multiple servers
+
+This example gets objects that represent the **Application** event logs on three computers:
+Server01, Server02, and Server03. The **ForEach** keyword is used because the **ComputerName**
+parameter accepts only one value. For more information, see [about_Foreach](../Microsoft.PowerShell.Core/about/about_Foreach).
+
+```PowerShell
```

Comment:
The languange tag should be all lowercase: 'powershell'

Reply:
Thanks, resolved.